### DISCUSSION TITLE
element是怎么样安全的升级依赖的啊

### DISCUSSION THREAD

**Discussion by TLovers** [element-plus/element-plus#9325] [Q&A] [answered]
想知道element是怎么升级package.json里面的依赖的？升级后又如何保证没有问题的

**Comment by sxzz** [comment, marked as answer]
使用 [renovate](https://github.com/renovatebot/renovate) https://github.com/element-plus/element-plus/issues/4096

每周会自动提 PR，如果 CI 测试通过了，那就没什么问题。

**Comment by baiwusanyu-c** [comment]
学到了
